bedrock/base/templates/includes/newrelic.html

```diff
@@ -0,0 +1,10 @@
+{% if settings.NEWRELIC_APP_ID %}
```

Comment:
How about we change this to
```
{% if settings.NEWRELIC_APP_ID and settings.NEWRELIC_BROWSER_LICENSE_KEY %}
```
since both are used in the snippet below?